client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -1784,4 +1792,132 @@ def create_external_build(self, location: str, requires_root: Optional[bool] = N
         :class:`~verta.endpoint.build.Build`
 
         """
-        return Build._create_external(self._conn, self.workspace, self.id, location, requires_root, scan_external, self_contained)
+        return Build._create_external(
+            self._conn,
+            self.workspace,
+            self.id,
+            location,
+            requires_root,
+            scan_external,
+            self_contained,
+        )
+
+    def finetune(
+        self,
+        destination_registered_model: Union[
+            str,
+            "verta.registry.entities.RegisteredModel",
+        ],
+        train_dataset: _dataset_version.DatasetVersion,
+        eval_dataset: Optional[_dataset_version.DatasetVersion] = None,
+        test_dataset: Optional[_dataset_version.DatasetVersion] = None,
```

Comment:
Not actually sure whether we want/need `test_dataset`. I included it because it felt reasonable for calculating end-of-training performance, but `eval_dataset` might just serve the same purpose.

Reply:
I say leave it 🤷🏼‍♀️